vault/activity_log.go

```diff
@@ -85,6 +86,16 @@ const (
 	nonEntityTokenActivityType = "non-entity-token"
 	entityActivityType         = "entity"
 	secretSyncActivityType     = "secret-sync"
+
+	// ActivityLogMinimumRetentionMonths sets the default minimum retention_months
+	// to enforce when reporting is enabled. Note that this value is also statically
+	// defined in the UI. Any updates here should also be made to
+	// ui/app/models/clients/config.js.
```

Comment:
We've updated this and we no longer set a default in the UI, so this comment can be removed. The UI will use whatever value is returned by the API. 

Thank you for thinking of the UI ☺️